### PR TITLE
Java doc improve post post manager

### PR DESCRIPTION
**Pull request by sebastianibam002**

### CODE REVIEW

File: src/main/java/use_cases/PostManager.java [hunk 1]
```diff
@@ -27,7 +33,7 @@ public PostManager(DatabaseManager databaseManager){
     public PostManager(ArrayList<Post> posts){
         this.databaseManager = new MySQLController();
         this.userManager = new UserManager(databaseManager);
-        Post[] allPosts = this.databaseManager.getAllPosts();
+        // Post[] allPosts = this.databaseManager.getAllPosts();
```
Comment (ericjest) [inline src/main/java/use_cases/PostManager.java:36]: if we don't need this line of code can we delete it?

**Review by ericjest — CHANGES_REQUESTED** [review]

File: src/main/java/entities/Post.java [hunk 2]
```diff
@@ -30,40 +39,82 @@ public boolean equals(Object o) {
...
     }
 
+    /**
+     * @return the id we assigned when we first created
+     * the Post object
+     */
     public String getId() { return this.id; }
 
+    /**
+     * @return return the time when it was posted
```
Comment (yowang20) [inline src/main/java/entities/Post.java:65]: Return the time Post was created

File: src/main/java/entities/Post.java [hunk 3]
```diff
@@ -30,40 +39,82 @@ public boolean equals(Object o) {
...
+     * saved in the object
+     */
     public LocalDateTime getTime(){
         return this.createdTime;
     }
+
+    /**
+     * given a Comment object it adds it stores the new
```
Comment (yowang20) [inline src/main/java/entities/Post.java:73]: Saves the comment to the Post's list of comments

File: src/main/java/entities/Post.java [hunk 4]
```diff
@@ -30,40 +39,82 @@ public boolean equals(Object o) {
...
+     * comment in the object
+     * @param comment stores the information about the
```
Comment (yowang20) [inline src/main/java/entities/Post.java:75]: the comment to be added to the post

File: src/main/java/entities/Post.java [hunk 5]
```diff
@@ -30,40 +39,82 @@ public boolean equals(Object o) {
...
+     *                comment
+     */
     public void addComment(Comment comment) {
         this.comments.add(comment);
     }
 
+    /**
+     * given the user that decided to like the post
```
Comment (yowang20) [inline src/main/java/entities/Post.java:83]: Adds the given user to the Post's list of liked users.

File: src/main/java/entities/Post.java [hunk 6]
```diff
@@ -30,40 +39,82 @@ public boolean equals(Object o) {
...
+     * it stores the user
+     * @param user user that gave like to the post
```
Comment (yowang20) [inline src/main/java/entities/Post.java:85]: The user that liked the post

File: src/main/java/entities/Post.java [hunk 7]
```diff
@@ -30,40 +39,82 @@ public boolean equals(Object o) {
...
+     */
     public void addLike(User user) {
         this.likedUsers.add(user);
     }
 
+    /**
+     * @return  all the
+     * users that have given like to the Post
```
Comment (yowang20) [inline src/main/java/entities/Post.java:93]: All the users that liked the Post

**Review by yowang20 — CHANGES_REQUESTED** [review]

**Review by ericjest — APPROVED** [review]

**Comment by ericjest** [comment]
@sebastianibam002 can you resolve the conflicts in your terminal?

**Comment by sebastianibam002** [comment]
Let me try

**Pull request merged**